clients/src/main/resources/common/message/FindCoordinatorRequest.json

```diff
@@ -23,12 +23,16 @@
   // Version 2 is the same as version 1.
   //
   // Version 3 is the first flexible version.
-  "validVersions": "0-3",
+  //
+  // Version 4 adds CoordinatorKeys
+  "validVersions": "0-4",
   "flexibleVersions": "3+",
   "fields": [
-    { "name": "Key", "type": "string", "versions": "0+",
+    { "name": "Key", "type": "string", "versions": "0-3",
       "about": "The coordinator key." },
     { "name": "KeyType", "type": "int8", "versions": "1+", "default": "0", "ignorable": false,
```

Comment:
Should we also change the `versions` for this to `0-3` rather than `1+`?

Reply:
`KeyType` was added in version 1 to resolve transaction coordinators. Before that, FindCoordinator was only able to resolve group coordinators.
It's still used in version 4 as we typically never want to resolve transaction and consumer coordinators at the same time so it determines the type of all keys in the request.

Reply:
Thanks for the explanation @mimaison, makes sense.